app/src/scripts/nav/navbar.jsx

```diff
@@ -11,9 +11,10 @@ import actions from '../user/user.actions.js'
 import { Navbar } from 'react-bootstrap'
 import { Modal } from '../utils/modal.jsx'
 import { refluxConnect } from '../utils/reflux'
-
 import brand_mark from './assets/brand_mark.png'
 
+import logo_app from './assets/logo_app.png'
```

Comment:
Looks like this an the related new asset are unused and can be removed.

Reply:
Updates have been pushed to fix these issues.